### PR TITLE
Feature/express to nest

### PR DESCRIPTION
**Pull request by Lunessr**

### CODE REVIEW

File: src/users/interfaces/user.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Role } from "src/enums/role";
```
Comment (rudenkose) [inline src/users/interfaces/user.ts:1]: same with path

File: src/users/users.factory.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Injectable } from "@nestjs/common";
+import { Role } from "src/enums/role";
```
Comment (rudenkose) [inline src/users/users.factory.ts:2]: 1 more

**Review by rudenkose — APPROVED** [review]
Looks good!
minor comments inside

File: src/authorization/authorization.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { CanActivate, ExecutionContext, Injectable } from "@nestjs/common";
+import * as jwt from "jsonwebtoken";
+import { Observable } from "rxjs";
+import { ERROR_MESSAGES } from "src/errors/errors";
+
+@Injectable()
```
[the remark this answers is not in the document]
Comment (Lunessr) [reply src/authorization/authorization.ts:6]: I think so, yes.

**Pull request merged**